src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -84,10 +84,28 @@
     @Option(doc = ReadStructure.PARAMETER_DOC, shortName = "RS")
     public ReadStructure READ_STRUCTURE;
 
+    @Option(doc = "Include skips and molecular barcodes for both phasing and pre-phasing values.  Use this if the either of the former are in-line in the read(s) (not in index reads).")
+    public boolean INCLUDE_SKIPS_AND_MOLECULAR_BARCODES = false;
```

Comment:
Ah, I thought I was going to skate by without writing tests, but now I see `IlluminaLaneMetricsCollectorTest`.  Shame on me.